pdaltools/las_info.py

```diff
@@ -19,6 +21,44 @@ def las_info_metadata(filename: str):
         raise RuntimeError(f"pdal info failed with error: \n {ret.stderr}")
 
 
+def las_info_from_header(filename: str):
+    r = pdal.Reader.las(filename=filename)
+    p = r.pipeline()
+    metadata = p.quickinfo["readers.las"]
+
+    return metadata
+
+
+def get_bounds_from_header_info(metadata):
+    bounds = metadata["bounds"]
+    minx, maxx, miny, maxy = bounds["minx"], bounds["maxx"], bounds["miny"], bounds["maxy"]
+
+    return minx, maxx, miny, maxy
+
+
+def get_epsg_from_header_info(metadata):
+    if "srs" not in metadata.keys():
+        raise RuntimeError(
+            "EPSG could not be inferred from metadata, please provide a 'proj' value in the 'color' method."
+        )
```

Comment:
Pb avec la fin: "please provide a proj value in the color method": Tu expliques comment utiliser, pas cette méthode, mais la méthode qui appelle cette methode. Je préfèrerais une précision sur la cause de l'erreur. 

`"EPSG could not be inferred from metadata: No key srs in metadata"`

Reply:
Effectivement, en plus ça permet d'être plus explicite avec 2 messages différents venant des 2 cas d'erreur différents